### PR TITLE
[Task-48][feat] Implement Base-Response class

### PR DESCRIPTION
**Pull request by OlegPanasyuk**
- It was needed because the response hadn't any template and was not similar.

- It provides the ability to create processing of all responses from the server.

- Also, it provides to manage custom status codes.

Trello ticket: Task-48: Base-Response: Back-End

### CODE REVIEW

File: 3.env [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PORT = 8080
```
Comment (melonholy) [inline 3.env]: REMOOOOOOVE IT FASTER

File: api/helpers/statusMessagesHelper.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    401: 'Unauthorized',
+    403: 'Forbidden',
+    404: 'Not Found',
+    405: 'Method Not Allowed',
+    406: 'Not Acceptable',
+    412: 'Precondition Failed',
+    415: 'Unsupported Media Type',
+    500: 'Internal Server Error',
+    501: 'Not Implemented',
+}
```
Comment (melonholy) [inline api/helpers/statusMessagesHelper.js:8]: Remove unused status codes, if we will need some status code we just will add him.

**Review by melonholy — CHANGES_REQUESTED** [review]

File: 3.env (hunk 1 above)
Comment (OlegPanasyuk) [reply 3.env]: Ok.

File: api/helpers/statusMessagesHelper.js (hunk 2 above)
Comment (OlegPanasyuk) [reply api/helpers/statusMessagesHelper.js]: This status is a base instance for HTTP protocol.

Comment (OlegPanasyuk) [reply api/helpers/statusMessagesHelper.js:8]: It just an example that you could to add some custom codes to your response.

File: 3.env (hunk 1 above)
Comment (OlegPanasyuk) [reply 3.env]: It's fucking .gitignore.  I have a rule in it `*.env`

**Review by melonholy — APPROVED** [review]

File: api/services/response/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const { statusMessagesHelper: statusMessages } = require('../../helpers');
+
+class BaseResponse {
+  getBody({ status = 200, data = {}, message = '' }) {
+    return {
+      status,
+      statusMessage: statusMessages[status],
+      message,
+      data,
+    };
```
Comment (slavasgod) [inline api/services/response/index.js:11]: What is the logical sense of "message" field? Maybe it's better to remove it and keep just 3 fields (status, statusMessage, data)?

Comment (slavasgod) [reply api/services/response/index.js:11]: And I also don't like name "getBody". This method transform the response and it looks like a selector. Maybe smth like: responseSelector, responseBuilder?

Comment (OlegPanasyuk) [reply api/services/response/index.js:11]: Look,  statusMessage is a short message and it depends on file in the system. Field Message is a long message which contains additional information about result of this event. Such a way I think.

Comment (OlegPanasyuk) [reply api/services/response/index.js:11]: DOne

**Review by slavasgod — APPROVED** [review]

**Pull request merged**